pkg/component/operator/web/v0/crawl_website.go

```diff
@@ -128,20 +156,36 @@ func (e *execution) CrawlWebsite(input *structpb.Struct) (*structpb.Struct, erro
 		title := util.ScrapeWebpageTitle(doc)
 		page.Title = title
 
-		defer mu.Unlock()
```

Comment:
Any reason why we don't use `defer` here?

Reply:
The colly is hard to understand.
So, I just tried lots of workaround to confirm if it works.

I will clean the code and make sure the performance is same.